crates/sui-protocol-config/src/lib.rs

```diff
@@ -1826,6 +1832,7 @@ impl ProtocolConfig {
                         cfg.feature_flags.narwhal_header_v2 = true;
                         cfg.feature_flags.random_beacon = true;
                         cfg.random_beacon_reduction_lower_bound = Some(1600);
+                        cfg.random_beacon_dkg_timeout_round = Some(200);
```

Comment:
can you change previous versions? (no conflict with the release snapshots)

Reply:
Yes if it's devnet-only